res/developer/builderConfig.json

```diff
@@ -1,12 +1,12 @@
 {
   "appId": "com.electron.brave",
-  "productName": "Brave-Developer",
+  "productName": "Brave Ads Trial",
   "npmRebuild": false,
   "mac": {
     "target": "dmg"
   },
   "dmg": {
-    "title": "Brave-Developer",
+    "title": "Brave Ads Trial",
```

Comment:
I think this needs to be `Brave-Ads-Trial`. If I understand correctly this is dmg file name.